static/js/util/courseList.js

```diff
@@ -23,11 +24,8 @@ function asPercent(number) {
 
 /**
  * Determine React elements for the UI given a course status
- * @param {object} course A course coming from the dashboard
- * @param {moment} now The current time
- * @returns {ReactElement} Some React element or string to display for course status
  */
-export function makeCourseStatusDisplay(course, now = moment()) {
+export function makeCourseStatusDisplay(course: Object, now: moment = moment()): string|React$Element {
```

Comment:
It would be nice to specify a type for course and run here though I'm happy doing that in another PR